src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Serialization.cs

```diff
@@ -153,6 +152,35 @@ public void WriteTo(ObjectWriter writer)
                     writer.WriteInt32(v);
                 }
             }
+
+            if (_simpleTypeNameToExtensionMethodMap == null)
```

Comment:
did you increment the serialization version?

Reply:
Nope, I didn't realize there's a version :P

Reply:
I think this would fix it
https://github.com/dotnet/roslyn/pull/39126/commits/216344dc14c382cda50452148b1695dff254c3a6

Reply:
Fixed now

Reply:
#resolved